staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -809,82 +815,87 @@ func (meal *cfgMeal) processOldPLsLocked() {
 // QueueSets.
 func (meal *cfgMeal) finishQueueSetReconfigsLocked() {
 	for plName, plState := range meal.newPLStates {
-		if plState.pl.Spec.Limited == nil {
+		if false && plState.pl.Spec.Limited == nil {
 			klog.V(5).Infof("Using exempt priority level %q: quiescing=%v", plName, plState.quiescing)
 			continue
 		}
-
-		limited := plState.pl.Spec.Limited
+		nominalConcurrencyShares, lendablePercent, borrowingLimitPercent := plSpecCommons(plState.pl)
```

Comment:
I'm wondering if we can move these refactorings to a separate PR to submit them faster and make the actual implementation PR simpler thanks to it.

Reply:
So the idea is to first do a PR that makes QueueSet support exempt behavior and makes the config-consuming controller use that, then follow with this PR that adds borrowing by exempt?